tsconfig.build.bundle.json

```diff
@@ -4,8 +4,8 @@
   "compilerOptions": {
     "paths": {
       "@prisma/client": ["./packages/client/src"],
-      "@prisma/sdk": ["./packages/sdk/src"],
-      "@prisma/sdk/dist/*": ["./packages/sdk/src/*"],
+      "@prisma/internals": ["./packages/sdk/src"],
+      "@prisma/internals/dist/*": ["./packages/sdk/src/*"],
```

Comment:
Uhm, maybe this change invalidates some other comments I highlighted above 🧐

Reply:
Indeed, I think it does (and CI is mostly 🟢 now)